setup/gcp/installer/forseti_installer.py

```diff
@@ -66,10 +84,21 @@ def run_setup(self):
                                                       bucket_name)
 
         # After deployment.
-        self.post_install_instructions(deploy_success,
-                                       deployment_name,
-                                       conf_file_path,
-                                       bucket_name)
+        instructions = self.post_install_instructions(deploy_success,
+                                                      deployment_name,
+                                                      conf_file_path,
+                                                      bucket_name)
+
+        if previous_instructions is not None:
+            instructions = previous_instructions + instructions
+
+        if last_installation:
```

Comment:
nit: last_installation isn't really descriptive. There's only one install?

Reply:
Updated to final_setup